client/src/overview/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Overview = (props) => {
```

Comment:
since all you use is `product_id` this can happily be destructured from the onset:

```node.js
const Overview = ({ product_id }) => { ...
```